src/components/input/UIInput.vue

```diff
@@ -118,11 +118,19 @@
         </div>
 
         <UIExpandTransition>
-            <slot v-if="error || $slots.error" name="error">
-                <p class="text-color-error text-sm">
-                    {{ error }}
-                </p>
-            </slot>
+            <div v-if="error || $slots.error || help || $slots.help">
```

Comment:
you introduced a redundant `<div>` here. You can use `<template>` which is only a logical block and won't render anything.